build-tools/check-boot-times/Program.cs

```diff
@@ -544,30 +544,30 @@ static string GetProgramPath (params string [] filenames)
 					if (!string.IsNullOrWhiteSpace (sdkPath)) {
 						potentialLocations.AddRange (new []{
 							$"{sdkPath}/platform-tools",
-							$"{sdkPath}/cmdline-tools/4.0/bin",
+							$"{sdkPath}/cmdline-tools/5.0/bin",
```

Comment:
Maybe this is a good opportunity to use value of the `$(CommandLineToolsVersion)` property here [a'la xaprepare](https://github.com/xamarin/xamarin-android/blob/4364b4ca94759acf637b75f95d62a5384852e69f/build-tools/xaprepare/xaprepare/xaprepare.targets#L36-L41)?

Reply:
In theory, yes.  In practice, I'm trying to get things ready for branching ASAP. :-)